apps/editor/src/js/wysiwygEditor.js

```diff
@@ -860,9 +858,9 @@ class WysiwygEditor {
       return result;
     });
 
-    // replace a space of the first and end in sapn tag to &nbsp;.
-    html = html.replace(FIND_OPENING_SPAN_WITH_SPACE, '<span$1>&nbsp;');
-    html = html.replace(FIND_CLOSING_SPAN_WITH_SPACE, '&nbsp;</span>');
+    // replace a space of the first and end in b, i, span to &nbsp;.
+    html = html.replace(/<(b|i|span)([^>]*)>[\u0020]/g, '<$1$2>&nbsp;');
+    html = html.replace(/[\u0020]<\/(b|i|span)>/g, '&nbsp;</$1>');
```

Comment:
It is our convention to define and use regular expressions in one constant. So I think this regular expression can change the constant value (`FIND_OPENING_SPAN_WITH_SPACE`, `FIND_CLOSING_SPAN_WITH_SPACE`) used in the past.